compiler/x/runtime/X86Runtime.hpp

```diff
@@ -66,6 +73,15 @@ inline bool jitGetCPUID(TR_X86CPUIDBuffer* pBuffer)
       // EAX = 7, ECX = 0
       cpuidex(CPUInfo, 7, 0);
       pBuffer->_featureFlags8 = CPUInfo[EBX];
+
+      // Check for XSAVE
+      if(pBuffer->_featureFlags2 & 0x08000000) // OSXSAVE
+         {
+         if((6 & _xgetbv(0) != 6) || feGetEnv("TR_DisableAVX")) // Unset OSXSAVE if not enabled via CR0
```

Comment:
Can you annotate the naked constant `6` with a comment describing what XCR bits it represents?  e.g.,
```
// '6' = mask for XCR0[2:1]='11b' (XMM state and YMM state are enabled)
```

Reply:
Good suggestion, updated.